aikit/automl/persistence/storage.py

```diff
@@ -0,0 +1,104 @@
+import os
+import json
+import pickle
+import pandas as pd
+from smart_open import open
+from aikit.tools.json_helper import SpecialJSONEncoder, SpecialJSONDecoder
+
+
+
+class Storage:
+
+    def __init__(self, path):
+        self.path = path
+        self.backend = FileStorage(path)
```

Comment:
so the idea is to use "smart_open" when the action needs at 'open' and a special 'backend' otherwise ?

Reply:
Not sure it still makes sense to manage 2 connections to the same storage.